### DISCUSSION TITLE
Tell me about your favorite Manga! あなたの好きな漫画を教えてください

### DISCUSSION THREAD

**Discussion by teilwal-dot** [NAIST-SE/AttractingContributors#489] [General]
面白い漫画を知りたいので、好きな漫画を教えて欲しいです。
ジャンル問わずなんでも読むので、気にせず送ってください！
I would like to know about your favorite manga.
I read anything in any genre.
So, don't worry about it, just send your favorite manga to me!

Here's my repo : https://github.com/teilwal-dot/AttractingContributors-Manga.git

**Comment by tmymzkr** [comment]
I done pull request 
please help me
https://github.com/m-tomoya/AttractingContributors-movie
